src/Backend/Modules/MediaLibrary/Installer/Data/locale.xml

```diff
@@ -266,6 +266,22 @@
         <translation language="en"><![CDATA[back to overview]]></translation>
         <translation language="nl"><![CDATA[terug naar overzicht]]></translation>
       </item>
+      <item type="label" name="Crop">
+        <translation language="en"><![CDATA[crop]]></translation>
+        <translation language="nl"><![CDATA[bijsnijden]]></translation>
+      </item>
+      <item type="label" name="RotateLeft">
+        <translation language="en"><![CDATA[rotate left]]></translation>
+        <translation language="nl"><![CDATA[draai naar links]]></translation>
+      </item>
+      <item type="label" name="RotateRight">
+        <translation language="en"><![CDATA[rotate right]]></translation>
+        <translation language="nl"><![CDATA[draai naar rechts]]></translation>
+      </item>
+      <item type="label" name="CropImage">
```

Comment:
This translation belongs to < Backend >< Core >, otherwise the label isn't visible in other modules.